### ISSUE TITLE
Increasing visibility of women scientists

### ISSUE THREAD

**Issue by kekoziar** [swcarpentry/python-novice-inflammation#1061]
Since the list `names = ['Curie', 'Darwin', 'Turing']` is redefined in [Episode 5](https://swcarpentry.github.io/python-novice-inflammation/05-loop.html), and there's no need to retain the previous names, I suggest replacing `Turing` with `Lovelace` (representing Ada Lovelace)
```
name = 'Rosalind'
for name in ['Curie', 'Darwin', 'Lovelace']:
    print(name)
print('after the loop, name is', name)
```

**Comment by edbennett** [comment]
Would the intention be to use a different list of names in different episodes? I was under the impression that the names were being kept the same between the two episodes to reduce the number of different lists that learners encounter, which may (slightly) reduce cognitive load.

If the plan is to remove Turing entirely, then I would point out that this would remove representation of LGBTQ+ scientists.

**Comment by kekoziar** [comment]
The intent is to increase representation of female scientists.  As is evidenced by this lesson, the vast majority of my life the only female scientist people can name is Marie Curie. It'd be nice to use others. Perhaps Ada Lovelace or Sally Ride (who was an astronaut, physicist, closeted gay from the public, and the original whistle-blower on the Challenger disaster).

If anything, I'd rather replace Darwin with Sally Ride to increase both female and LGBTQ+ representation.

**Comment by ErinBecker** [comment]
FWIW I agree with replacing Darwin with Sally Ride.

**Comment by edbennett** [comment]
If the name list is kept consistent between the episodes that use `names` lists, then I would support replacing Darwin with Sally Ride as well (with or without replacing Turing with Ada Lovelace).

Do you think full names should be used rather than surnames? `Lovelace` would be easily searchable for the unfamiliar, but `Ride` less so.

**Comment by kekoziar** [comment]
> If the name list is kept consistent between the episodes that use `names` lists, then I would support replacing Darwin with Sally Ride as well (with or without replacing Turing with Ada Lovelace).
> 
> Do you think full names should be used rather than surnames? `Lovelace` would be easily searchable for the unfamiliar, but `Ride` less so.

Yes, I was thinking about that. I don't think the extra names wouldn't make typing too onerous. `['Marie Curie', 'Alan Turing', 'Sally Ride']`, then in the `for` loop in Episode 5 we could use `Rosalind Franklin` instead of only her first name.
